extensions/ql-vscode/src/model-editor/modeling-store.ts

```diff
@@ -344,6 +344,20 @@ export class ModelingStore extends DisposableObject {
     });
   }
 
+  public getProcessedByAutoModelMethods(
+    dbItem: DatabaseItem,
+    methodSignatures?: string[],
+  ): Set<string> {
+    const processedByAutoModelMethods =
+      this.getState(dbItem).processedByAutoModelMethods;
+    if (!methodSignatures) {
+      return processedByAutoModelMethods;
+    }
+    return new Set(
+      Array.from(processedByAutoModelMethods).filter(methodSignatures.includes),
```

Comment:
Unfortunately using `filter(methodSignatures.includes)` is causing an error. I thought it would be ok since it's a local variable 😢 

Converting to an anonymous function gets everything working.

Reply:
It might be calling `Array.includes` with the wrong `this` parameter, so `methodSignatures.includes.bind(this)` will probably work. I do think an anonymous function is clearer here.